lib/auth/auth.go

```diff
@@ -5371,6 +5380,79 @@ func (a *Server) syncDynamicLabelsAlert(ctx context.Context) {
 	}
 }
 
+func (a *Server) cleanupNotifications(ctx context.Context) {
+	userNotifications, _, err := a.Cache.ListUserNotifications(ctx, 0, "")
+	if err != nil {
+		slog.WarnContext(ctx, "failed to list first set of notifications for periodic cleanup", err)
```

Comment:
```suggestion
		slog.WarnContext(ctx, "failed to list first set of notifications for periodic cleanup", "error", err)
```

After the initial message, `slog` logs key/value pairs, and convention is to use the key `error` for errors.